app/graphql/albums/fieldResolvers.js

```diff
@@ -0,0 +1,15 @@
+const { gql } = require('apollo-server');
+const albumsService = require('../../services/album');
+
+module.exports = {
+  fieldResolvers: {
+    photos: obj => albumsService.getPhotosOfAlbum(obj.id),
+    artist: obj => albumsService.getArtistOfAlbum(obj.userId)
```

Comment:
artist wont be necessary for this training

Reply:
Should I remove it?

Reply:
yes!